app.drey.EarTag.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id" : "app.drey.EarTag",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "eartag",
+    "finish-args" : [
+        "--filesystem=xdg-pictures:ro",
+        "--filesystem=xdg-music:rw",
+        "--filesystem=xdg-download:rw",
```

Comment:
Are you sure your app does not use a portal already?

Reply:
If I understand the docs correctly, it *should* be using a portal becuase I'm using GtkFileChooserNative, but for some reason it wasn't doing so on my system when I initially tested it.